meta/regular_expression.rb

```diff
@@ -0,0 +1,14 @@
+Mutant::Meta::Example.add :'regexp_expression' do
+  source '/(foo|bar)/'
```

Comment:
This is IMO a more complex example than needed for the first iteration, as it contains an OR and a grouping, lets reduce it to `/foo|bar/` to focus on only one thing?